### ISSUE TITLE
How to allow Vendor ID & Product ID are zero

### ISSUE THREAD

**Issue by TshuW** [tekezo/Karabiner-Elements#1862]
https://github.com/tekezo/Karabiner/issues/403
https://github.com/tekezo/Karabiner-Elements/issues/163
I have the similar issue on my bluetooth keyboard, so I wander is there any way to allow vendor ID & product ID are zero on Keyboard Maestro

**Comment by Voleking** [comment]
Sorry, I haven't read the manual carefully, karabiner-element can also use "location_id" to specify device. You can examine them in EventViewer > Devices tab.

**Comment by Voleking** [comment]
however, seems like karabiner-element cannot specify device only with `location_id`,
for example, the following condition doesn't work on the latest beta vesion:
```json
                    "conditions": [
                        {
                            "identifiers": [
                                {
                                    "description": "HHKB BLE",
                                    "is_keyboard": true,
                                    "is_pointing_device": true,
                                    "location_id": 107278329
                                }
                            ],
                            "type": "device_unless"
                        }
                    ]
```

**Comment by Voleking** [comment]
active

**Comment by zenof** [comment]
same issue here

**Comment by chriskisskiss** [comment]
same issue

**Comment by tshu-w** [comment]
@tekezo Should we reopen this issue?

**Comment by 1-WEEK** [comment]
same issue

**Comment by joonhyuk** [comment]
same issue

**Comment by fabiomcosta** [comment]
Same issue :/

**Comment by GXFG** [comment]
+1

**Comment by tshu-w** [comment]
Seems like this issue is fix on 14.10
https://karabiner-elements.pqrs.org/docs/releasenotes/

**Comment by stackia** [comment]
```
{
  "location_id": 229706159,
  "is_keyboard": true
}
```
This still not working on 14.12.0
